runtime/onert/core/src/exec/Executors.cc

```diff
@@ -97,99 +97,64 @@ void Executors::checkSupportedMultimodel() const
   //  (n)th model: output assumption
   //  Otherwise: edges assumption
 
-  // Assumption: executor input/output size
-  //  Models size: n
-  //  sizeof(model(x).outputs) == sizeof(model(x+1).inputs) if 0 < x < n-2
-  //  sizeof(model(0).inputs) == sizeof(package.inputs)
-  //  sizeof(moddel(n-1).outputs) == sizeof(package.outputs)
-  for (auto &pair : _executors)
-  {
-    auto const &model_index = pair.first.first;
-    auto const &subg_index = pair.first.second;
-
-    // Skip inner subgraph
-    if (subg_index != ir::SubgraphIndex{0})
-      continue;
-
-    auto executor = at(model_index, subg_index);
-
-    // Last model's output
-    if (model_index.value() == model_count - 1)
-    {
-      if (executor->graph().getOutputs().size() != _model_edges->pkg_outputs.size())
-        throw std::runtime_error{"NYI: Unsupported package output pattern"};
-
-      continue;
-    }
-
-    // 1st model's input
-    if ((model_index == ir::ModelIndex{0}) &&
-        (executor->graph().getInputs().size() != _model_edges->pkg_inputs.size()))
-      throw std::runtime_error{"NYI: Unsupported package input pattern"};
-
-    auto const next_index = ir::ModelIndex{static_cast<uint16_t>(model_index.value() + 1)};
-    auto const next_executor = at(next_index, ir::SubgraphIndex{0});
-
-    if (executor->graph().getOutputs().size() != next_executor->graph().getInputs().size())
-      throw std::runtime_error{"NYI: Multi model execution for this package is not supported yet"};
-
-    // Check model output is connected to other model
-    for (uint32_t i = 0; i < executor->graph().getOutputs().size(); i++)
-    {
-      auto const &edges = _model_edges->edges;
-      auto check_from_edge = [&](const ir::ModelEdge &edge) {
-        return (std::get<ir::ModelIndex>(edge.from) == model_index) &&
-               (std::get<ir::SubgraphIndex>(edge.from) == subg_index) &&
-               (std::get<ir::IOIndex>(edge.from).value() == i);
-      };
-
-      if (std::find_if(edges.begin(), edges.end(), check_from_edge) == edges.end())
-      {
-        throw std::runtime_error{"NYI: Unsupported model edge pattern"};
-      }
-    }
-  }
-
   // Assumption: edges
-  //   x:0:z -> x+1:0:z  (0 < x < n-2)
+  //   x:0:z -> x+1:0:z'
```

Comment:
```suggestion
  //   x:0:z -> x+1:0:z
```

Reply:
`from.IOIndex` and `to.IOIndex` can be different on this commit.